content/src/fieldConfig/profile.json

```diff
@@ -164,7 +164,8 @@
           "header": "New Jersey Tax ID",
           "placeholder": "Enter New Jersey Tax ID number",
           "description": "What is your `New Jersey Tax ID?|tax-id`",
-          "errorTextRequired": "Enter a 12-digit NJ Tax ID"
+          "errorTextRequired": "Enter the 12-digit Tax ID shown in your tax confirmation email.",
```

Comment:
I think this is a rebase issue. The old text "Enter a 12-digit NJ Tax ID" is what should be here